server/.env

```diff
@@ -1,2 +1,4 @@
-TEAM_NAME="Shums"
+TEAM_NAME="Werewolf"
 FLASK_ENV="development"
+DB_SERVER="cluster0.gjq25.mongodb.net"
+DB_NAME="team-werewolf"
```

Comment:
I thought we agreed to get rid of this? I currently have a db package that has the server name setup as a global. Prob not that big a deal either way though

Reply:
yeah we definitely don't have to keep this - without a db package in place I was trying to find a logically sound way to store this info

Reply:
that's totally fair. I've removed it in my branch so if no one else has a reason to keep it then I think we're good